src/modules/TeamsList/components/TeamListModals/index.tsx

```diff
@@ -180,15 +191,19 @@ export const TeamListModals: FC<{ page: number }> = ({ page }) => {
     updateTeam();
   };
 
+  const onSubmitSortStudents = () => {
+    sortStudents();
+  };
+
   return (
     <>
       <ModalExpel
-        title="Leave Team"
+        title="Leave team"
```

Comment:
translation?

Reply:
all the charm of i18n that is I can just add translation for all coming args in common componets like button